data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -22,34 +22,35 @@ class TokenSettings(ZkLendSpecificTokenSettings, TokenSettings):
     pass
 
 
-def convert_string_to_decimal_1e27(string_big_num):
+def format_reserve_data_number(string_large_num: str):
     """
-    Method to convert big string number to decimal.
+    Method to convert large string number to decimal.
```

Comment:
```suggestion
    Method to convert string number to decimal divided by the scale factor.
```
In general, this function can take small numbers as well 😉

Reply:
fix it in this commit, i change the comment 73f712a53242c2402dbad84e335c8b84ae1e3ebf